scripts/sdk-coin-generator/template/boilerplates/.testnet.ts

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
I think the spacing is off on this file. After generating, prettier shows the created testnet file with a leading space on every line.